develop/reference/articles/liferay-npm-bundler/08-changes-between-bundler-1.x-and-2.x.markdown

```diff
@@ -0,0 +1,53 @@
+# What Changed Between Liferay npm Bundler 1.x and 2.x
```

Comment:
This section of the reference looks quite good 👍 . Maybe you can link my previous review comment https://github.com/izaera/liferay-docs/pull/8/files#diff-cabbea5f40fbf26633f51fe21ec8f789R102 to here instead of the blog post 🤔

Reply:
See comment above